storage-node/src/services/logger.ts

```diff
@@ -82,16 +85,27 @@ const proxy = new Proxy(InnerLogger, {
 export default proxy
 
 /**
- * Creates Express-Winston logger options.
+ * Creates Express-Winston default logger options.
  *
  */
-export function createExpressLoggerOptions(): expressWinston.LoggerOptions {
+export function createExpressDefaultLoggerOptions(): expressWinston.LoggerOptions {
   return {
     winstonInstance: proxy,
     level: 'http',
   }
 }
 
+/**
+ * Creates Express-Winston error logger options.
+ *
+ */
+export function createExpressErrorLoggerOptions(): expressWinston.LoggerOptions {
+  return {
+    winstonInstance: proxy,
+    level: 'error',
```

Comment:
After some testing I suggest making a small change to this config. It seems possible to include some additional information in the error message (like `err.message` or `res.statusCode`) [by setting the value of `msg` config parameter](https://www.npmjs.com/package/express-winston/v/2.6.0#options-1). The default value for `msg` seems to be `middlewareError`, which doesn't say much about the error when you go through the logs in console / elasticsearch:
![image](https://user-images.githubusercontent.com/19978260/147939095-696edf95-d333-4129-a317-9a2bb9903295.png)
![image](https://user-images.githubusercontent.com/19978260/147939156-4523ed76-d224-4ebe-a0d8-280c1a696cde.png)